make.sh

```diff
@@ -1,8 +1,9 @@
-#! /bin/bash
+#!/usr/bin/env bash
```

Comment:
Are u sure u want this to be merged to master?
Wdy think @michael-schwarz

Reply:
I asked during one meeting if this change was ok. Without it I can't run the scripts on my system. 
@michael-schwarz I could put this change in a separate pull request if you want.

Reply:
Yes, putting these into a separate PR would clarify things here :+1:

Reply:
Pulled out into #276